circuit/js/test/unit/scaffold.test.ts

```diff
@@ -20,7 +21,9 @@ describe("Scaffold", () => {
     };
     const _artifact = await testCircuit.compile(defaultInputs);
     const computeQuery = await testCircuit.run(defaultInputs);
-    expect(computeQuery.vkey[0]).toEqual("0x0001000000010100000004010000010080000000000000000000000000000000");
+
+    // numValuesPerInstanceColumn: 2 * 7 + 16 * 128 = 2062 (0x80E)
+    expect(computeQuery.vkey[0]).toEqual("0x00010000080e0100000004010000010080000000000000000000000000000000");
```

Comment:
the answer here should match the one in the encoder test - the metadata does not depend on `resultLen` (see above).